app/views/admin/clubs/show.html.haml

```diff
@@ -0,0 +1,15 @@
+%p#notice= notice
+
+%p
+  %b Name:
+  = @club.name
+%p
+  %b Description:
+  = @club.description
+%p
+  %b Logo:
+  = image_tag @club.logo, size: '100x150'
```

Comment:
instead of setting size all the time, there must be an option in carrierwave to generate thumbnails automatically. please check that